agent/background.go

```diff
@@ -28,16 +26,15 @@ func (a *Agent) startHeartbeat(ctx context.Context, cancel context.CancelFunc, t
 		select {
 		case <-ticker.C:
 			signalBeat, err = a.doHeartbeat(ctx, tc)
-			if signalBeat == evergreen.TaskConflict {
-				tc.logger.Task().Error("Encountered task conflict while checking heartbeat, aborting task")
+			if signalBeat == evergreen.TaskFailed {
+				tc.logger.Task().Error("Received termination signal from app server, aborting task")
```

Comment:
This might be confusing since it doesn't seem like the agent is actually terminating, it's just being told to abort the task. Can this be rephrased as `"Heartbeat received signal to abort task"`?